tests/unit/test_interpolation_tables.py

```diff
@@ -7,6 +7,7 @@
 
 
 class _LinearFunc:
+    # Helper class to represent a linear function.
     def __init__(self, nd, *args):
         self.coeff = np.random.rand(nd, 1)
         self.tol = 1e-10
```

Comment:
The tolerances (`self.tol`,`self.diff_tol`) in _LinearFunc and _QuadraticFunc are parameters for the test and not attributes of a function. Why this choice is appropriated?